### DISCUSSION TITLE
Level Requirements for Gear

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/Prophets-of-the-Labyrinth#486] [Ideas]
In addition to "limited useabilty dropped gear", having one of the Fighter's gear require a certain level to be usable would be a way to add texture to the leveling process.

**Comment by doubledh** [comment]
would it be strictly superior than non-req gear? would it be a bonus or a strict can/t use at all, boundary?

**Reply by ntseng** [reply]
Originally imagined as "can't use until level 10", level requirements categorize as a big enough trade-off that I think it's ok to print a gear piece that would otherwise have strictly superior value to the players as it would create a pattern of "less efficient gear for early phases, try to get level required gear later". I say "ok" because it'd probably be more exciting (and less math obfuscated for players) if we explore other design space.

Beyond all that, level requirement gear didn't make it into the gear overhaul based on number of slots in the gear pool. We might need a focus mechanic to expand the pool further without reducing reliability of access to desired mechanics.
